diff --git a/pymammotion/client.py b/pymammotion/client.py
index 0000000..0000000 100644
--- a/pymammotion/client.py
+++ b/pymammotion/client.py
@@ -1296,7 +1296,7 @@ class MammotionClient:
             f"/sys/{record.product_key}/{record.device_name}/thing/event/+/post",
             f"/sys/proto/{record.product_key}/{record.device_name}/thing/event/+/post",
             f"/sys/{record.product_key}/{record.device_name}/app/down/thing/status",
-            # f"/sys/{record.product_key}/{record.device_name}/app/down/thing/properties",
+            f"/sys/{record.product_key}/{record.device_name}/app/down/thing/properties",
         ):
             transport.add_topic(topic)
         transport.register_device(record.product_key, record.device_name, iot_id)
